### PR TITLE
[PR] Update theme with child theme skeleton files

### PR DESCRIPTION
**Pull request by jeremyfelt**

### CODE REVIEW

**Comment by jeremyfelt** [comment]
@philcable #reviewmerge?

File: functions.php [hunk 1]
```diff
@@ -1,8 +1,203 @@
...
+ * @since 0.1.0
+ *
+ * @param string $post_type
+ */
+function wsu_alert_add_meta_boxes( $post_type ) {
+	if ( 'post' !== $post_type ) {
+		return;
+	}
+
+	add_meta_box( 'wsu_alert_active_stats', 'Alert Status', 'wsu_alert_display_alert_status', 'post', 'side', 'high' );
```
Comment (philcable) [inline functions.php]: I imagine `wsu_alert_active_stats` is meant to be `wsu_alert_active_status`, but it's not hurting anything the way it is.

File: functions.php [hunk 2]
```diff
@@ -1,8 +1,203 @@
...
+function wsu_alert_save_post( $post_id, $post  ) {
+	if ( defined( 'DOING_AUTOSAVE' ) && DOING_AUTOSAVE ) {
+		return;
+	}
+
+	if ( 'auto-draft' === $post->post_status ) {
+		return;
+	}
+
+	// @codingStandardsIgnoreStart
```
Comment (philcable) [inline functions.php]: Is this because we're not verifying a nonce?

File: parts/headers.php [hunk 3]
```diff
@@ -1,10 +1,41 @@
...
+
+				$alert_location = get_post_meta( $active_alert, 'wsu_alert_location', true );
+				if ( 'pullman' === $alert_location ) {
+					$alert_location = 'Pullman Campus';
+				} else {
+					$alert_location = 'All WSU';
+				}
+				?>
+				<header><?php echo esc_html( $alert_location ); ?></header>
+				<h1><?php echo esc_html( get_the_title( $active_alert ) ); ?></h1>
```
Comment (philcable) [inline parts/headers.php]: I would expect the title to be a link, too - or even instead of the date - but perhaps there's a reason it's not.

Comment (jeremyfelt) [inline parts/headers.php]: Yeah, I'm not entirely sure about that yet. The original mocks didn't have a title, so I'm kind of guessing too. I'm going to leave it as is for now and then adjust later once we get some feedback.

File: functions.php (hunk 2 above)
Comment (jeremyfelt) [inline functions.php]: Yep, I'm being a little lazy. Good catch :)

**Pull request merged**